packages/Confirmation/src/Confirmation.js

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Button from "@paprika/button";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import Popover from "@paprika/popover";
+import contentStyles from "./ContentStyles.styles";
+import ConfirmationStyles from "./Confirmation.styles";
+
+const propTypes = {
+  buttonSize: PropTypes.oneOf(ShirtSizes.DEFAULT),
+  confirmButtonType: PropTypes.oneOf(["primary", "destructive"]),
+  confirmLabel: PropTypes.string.isRequired,
```

Comment:
this must be node, we shouldn't restrict the user to be an only string value.

Reply:
This is the label for the button.  I'm not so sure it should be anything besides a plain string.